serum-390/soen390-team16-godzilla
# Authorization Frontend: per-user role and access level management

# Description

Adds the front end React component used to manage access levels and roles for users of the system.

## Link to work item

Related work items #177 (***not closed by this pr***)

## How to Test

1. Run the app
2. Click on settings > user management
3. Explore the user cards that are display, check out the side bar list (search bar functional)
4. Click on a user card to display the edit-user view; in this view new roles can be added to the user account and the user can be made into an admin
